Documentation/docs-mobile/messages/xa1038.md

```diff
@@ -0,0 +1,21 @@
+---
+title: .NET for Android error XA1038
+description: XA1038 error code
+ms.date: 06/27/2024
+---
+# .NET for Android error XA1038
+
+## Example messages
+
+```
+The 'AndroidResourceDesignerClassModifier' MSBuild property has an invalid value of 'foo'. Value values are 'public/private/internal'.
```

Comment:
"Value values are '…'" looks weird.

Firstly, it should be "*Valid* values", not "Value values".

Secondly, I'm not sure how clear `'public/private/internal'` will be, as I'm not sure how to interpret the intermixing of `/` and the quotes around the values.

"Better" would be: "Valid values are 'public', 'private', or 'internal'."  Except that's rather English-specific.

Perhaps we could go with: `A valid value is one of: public, private, internal.` or `A valid value is one of: 'public', 'private', 'internal'.`